django/contrib/admin/templates/admin/filter.html

```diff
@@ -1,8 +1,19 @@
 {% load i18n %}
-<h3>{% blocktranslate with filter_title=title %} By {{ filter_title }} {% endblocktranslate %}</h3>
+
+{% with field=title.split|join:"_" %}
+<a href="#" data-toggle="{{ field }}">
+    <h3>
+        <div data-expanded="{{ field }}">-</div>
+        <div data-collapsed="{{ field }}">+</div>
```

Comment:
Can we add a single `div` and switch a text?

Reply:
I think not because I need data-[expanded/collapsed] separately